addon/components/ember-thead/component.js

```diff
@@ -87,7 +87,7 @@ export default Component.extend({
   sorts: defaultTo(() => []),
 
   /**
-    An optional sort. If not specified, defaults to <sortMultiple>, which
+    An optional sort. If not specified, defaults to `<sortMultiple>`, which
```

Comment:
While typing documented arguments, I noticed that `<sortMultiple>` is not rendered in the API docs without the backticks